### PR TITLE
Transmission with UI fix for DSM 6.1 and ffmpeg with Intel Quick Sync

### PR DESCRIPTION
**Pull request by jvega1976**
I have made some updates to fix an issue with the integration of Transmission in DSM 6.1 (icon was missing).

I also have updated it to use the "Transmission Web Control" app as default UI.

For ffmpeg, I have compiled it with options --enable-libmfx and --enable-libvaapi to activate the Intel Quick Sync Hardware Acceleration.

Both tested using the apollolake-6.1 toolchain (DS218)

### CODE REVIEW

**Comment by ymartin59** [comment]
Please provide two seperate PRs targetting `master` for each commit for `ffmpeg` and `transmission` packages

File: cross/ffmpeg/Makefile [hunk 1]
```diff
@@ -31,6 +31,7 @@ CONFIGURE_ARGS += --enable-libtheora --enable-libvo-amrwbenc --enable-libvorbis
 CONFIGURE_ARGS += --enable-libvpx --enable-libx264 --enable-libx265 --enable-openssl
 CONFIGURE_ARGS += --disable-debug --disable-doc --disable-htmlpages --disable-manpages
 CONFIGURE_ARGS += --disable-podpages --disable-static --disable-txtpages
+CONFIGURE_ARGS += --disable-ffplay --disable-ffserver
```
Comment (cytec) [inline cross/ffmpeg/Makefile]: why disable those tools? i use ffserver frequently ;)

File: cross/libdrm/PLIST [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+lib/pkgconfig/libkms.pc
+share/man/man3/drmAvailable.3
+share/man/man3/drmHandleEvent.3
+share/man/man3/drmModeGetResources.3
+share/man/man7/drm.7
+share/man/man7/drm-gem.7
+share/man/man7/drm-kms.7
+share/man/man7/drm-memory.7
+share/man/man7/drm-mm.7
+share/man/man7/drm-ttm.7
```
Comment (cytec) [inline cross/libdrm/PLIST]: Please remove man pages and other stuff not needed and prefix the rest according to https://github.com/SynoCommunity/spksrc/wiki/PLIST-files#example

File: cross/libva/PLIST [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+lib/libva.so
+lib/libva.so.1
+lib/libva.so.1.9900.0
+lib/libva-tpi.la
+lib/libva-tpi.so
+lib/libva-tpi.so.1
+lib/libva-tpi.so.1.9900.0
+lib/pkgconfig/libva-drm.pc
+lib/pkgconfig/libva.pc
+lib/pkgconfig/libva-tpi.pc
```
Comment (cytec) [inline cross/libva/PLIST]: same as here

File: cross/mfx_dispatch/PLIST [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+include/mfxplugin++.h
+include/mfx_plugin_hive.h
+include/mfxsession.h
+include/mfxstructures.h
+include/mfx_vector.h
+include/mfxvideo.h
+include/mfxvideo++.h
+include/mfxvp8.h
+include/mfxvstructures.h
+include/mfx_win_reg_key.h
```
Comment (cytec) [inline cross/mfx_dispatch/PLIST]: and here

File: cross/openssl/Makefile.rej [hunk 5]
```diff
@@ -0,0 +1,8 @@
+diff a/cross/openssl/Makefile b/cross/openssl/Makefile	(rejected hunks)
+@@ -1,5 +1,5 @@
+ PKG_NAME = openssl
+-PKG_VERS = 1.0.2l
++PKG_VERS = 1.0.2m
+ PKG_EXT = tar.gz
+ PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+ PKG_DIST_SITE = ftp://ftp.openssl.org/source
```
Comment (cytec) [inline cross/openssl/Makefile.rej]: delete this file, seems to be a leftover?

File: cross/pciaccess/PLIST [hunk 6]
```diff
@@ -0,0 +1,7 @@
+include/pciaccess.h
+lib/libpciaccess.a
+lib/libpciaccess.la
+lib/libpciaccess.so
+lib/libpciaccess.so.0
+lib/libpciaccess.so.0.11.1
+lib/pkgconfig/pciaccess.pc
```
Comment (cytec) [inline cross/pciaccess/PLIST]: PLIST again

File: cross/pthread-stubs/Makefile [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+HOMEPAGE = http://www.intel.com
+COMMENT  = 
+LICENSE  = GPL
+
+GNU_CONFIGURE = 1
+
+YASM_PATH = $(WORK_DIR)/../../../native/yasm/work-native/install/usr/local/bin
+
+CONFIGURE_ARGS += CPP="/root/spksrc/toolchains/syno-apollolake-6.1/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-cpp"
```
Comment (cytec) [inline cross/pthread-stubs/Makefile]: Please don't use absolute links... use variables, since this will not work for anyone else!

**Comment by cytec** [comment]
Before opening a new PR for those Packages you should make sure you've read: https://github.com/SynoCommunity/spksrc/wiki/PLIST-files#example and adjust ALL PLIST files accordingly. Remove ABSOLUTE paths! That's not going to work for anyone else besides you, and most of the variables should already be set by spksrc... also please make sure that you'll only include the files that are needed for the package itself in your PR... 

There are changed files for sabnzbd, rutorrent, tvheadend etc in this PR...

**Pull request closed without merging**